runtime-manager/modules/ROOT/nav.adoc

```diff
@@ -1,5 +1,6 @@
 .xref:index.adoc[Runtime Manager]
 * xref:index.adoc[Anypoint Runtime Manager]
+//* xref:runtime-manager-release-notes.adoc[Release Notes]
```

Comment:
Is this commented out because it's in flux?

Reply:
I commented it out so that the RTM RNs do not become visible until we have the new RTM RNs page, as discussed.